frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/config.ts

```diff
@@ -4,7 +4,7 @@ export function getDefaultDisplayOptions(
   shouldShownDownloadData: boolean,
 ): EmbeddingDisplayOptions {
   return {
-    font: null,
+    font: "",
```

Comment:
We are already using `""` in the options. Using `null` crashes the Mantine v7's dropdown select as it expects a string.